src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -4073,7 +4073,7 @@ void M()
     {
         int a = 1;
         int {|Rename:y1|} = C.y;
-        var t = (a, x: y1);
+        var t = (a: a, x: y1);
```

Comment:
I'm a bit confused, do we have a setting somewhere that allows user's to specify that they want "a: a" vs. "a"? That seems weird to me.